core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/admin/ConfiguredToolingsRestService.java

```diff
@@ -42,7 +43,11 @@ public class ConfiguredToolingsRestService extends AbstractRestService {
 		description = "Provides a set of all available toolings ids configured on the server."
 	)
 	@RequestMapping(method = { RequestMethod.GET }, produces = { AbstractRestService.JSON_MEDIA_TYPE })
-	public @ResponseBody List<Terminology> toolings() {
-		return TerminologyRegistry.INSTANCE.getConfiguredTerminologies();
+	public @ResponseBody ResponseEntity<List<String>> toolings() {
```

Comment:
No need for the ResponseEntity wrapping, but we need a wrapping object instead see the example JSON I have sent you.